Add HeroBanner component with updated styles

SousthabChakma opened the pull request:
1. Created HeroBanner component in src/components/organisms/HeroBanner.tsx
2. Implemented responsive layout with centered text, styled button and updated globals.css
3. Updated page.tsx to include HeroBanner component

nabarunchakma commented:
@SousthabChakma  your PR has conflict.  Please resolve them.

nabarunchakma commented on src/app/globals.css:
```diff
@@ -30,4 +30,8 @@ body {
     text-wrap: balance;
   }
+
+  .bg-tan-silver {
```
@SousthabChakma  We might remove this file `globals.css`.  Please use tailwind css to style your component.

nabarunchakma commented on src/components/organisms/HeroBanner.tsx:
```diff
@@ -0,0 +1,27 @@
+        <h1 className="text-5xl font-semibold mb-6 text-left">
+          disciplined mind brings happiness
+        </h1>
+        <button className="bg-tan-silver text-black px-6 py-3 rounded-full text-lg font-semibold hover:bg-opacity-90 transition-all">
```
@SousthabChakma , there is a button done by @Devjoy-chakma .  You can reuse it here.  This will make it easy to maintain the button style everywhere with a single source.

SousthabChakma commented:
1.  Resolved the PR conflict (yarn.lock conflict)
2. Updated Herobanner by reusing the button by Devjoy Chakma and also using tailwind css to style the component
3. Updated page.tsx and tailwind.config.ts
